core/mvc/controllers/RestController.php

```diff
@@ -14,6 +14,7 @@
 use Gaia\Libraries\Utils\Util;
 use Phalcon\Events\ManagerInterface as EventsManagerInterface;
 use function Gaia\Libraries\Utils\create_guid;
+use phpDocumentor\Parser\Exception;
```

Comment:
Why is this required?

Reply:
Removed! It was a typo mistake by VSCODE.